stage/services/salesforce-api/salesforce-api_runner.js

```diff
@@ -160,9 +186,9 @@ exports.handler = async function (event, context) {
 
     // some member types support billing information
     if (
-      ["Direct Member", "Consortium Organization", "Member Only"].includes(
-        res.attributes.member_type
-      )
+      ["Direct Member", "Consortium Organization", "Member Only"].includes(res.attributes.member_type) &&
+      hasBillingInfo(res) &&
+      hasJoinedDate(res)
     ) {
       body = Object.assign(body, {
         Billing_Organization__c: res.attributes.billing_organization,
```

Comment:
Depending on the salesforce API could we just not check individual data like:
Billing_Organization__c: res.attributes.billing_organization ? res.attributes.billing_organization : null

Reply:
I am going to leave this the way it is for now.  I think checking individually will still allow billing info in salesforce to be nulled out if all fields are null.